Pearson-Advance/frontend-app-instructors-portal
# feat: Redux config

# Ticket 
https://agile-jira.pearson.com/browse/PADV-1146


## Description
In this PR is added the Redux config to get institutions, the first institution is displayed using `JSON.stringify`

## Additional changes 

![1](https://github.com/Pearson-Advance/frontend-app-instructors-portal/assets/52179095/0f02b560-3107-4663-8368-fdea77bec0cf)


## How to test
- Clone the instructor Portal MFE
- Run openedx services, make sure to configure a site in the port 1990 with the proper settings.
- Run 
```bash
npm install
```
- Then
```bash
npm run start
```
- Go to [http://http://localhost:1990](http://http//localhost:1990) you should be able to see the first institution info
